### ISSUE TITLE
USGS updated to https

### ISSUE THREAD

**Issue by Kaylen-Pillay** [udacity/ud843_Soonami#19]
Hey guys!

The USGS_REQUEST_URL has changed to https. This means that the current request url string will result in a HTTP 301 error response code (Moved permanently). Changing the request url from "http:// ... " to "https:// ... " fixes this.

Response codes 200!!!! Yay

**Comment by sahs228** [comment]
Yeah, I had problem in showing anything. I got 301 error, which said that the url is invalid.
But all i had to do to fix it was to add "s" after "http" to make it work.
